test/tokengenerator_test.go

```diff
@@ -0,0 +1,147 @@
+package testutils
+
+import (
+	"bytes"
+	"crypto/rsa"
+	"encoding/json"
+	"net/http"
+	"testing"
+
+	"github.com/dgrijalva/jwt-go"
+	uuid "github.com/satori/go.uuid"
+	"github.com/stretchr/testify/require"
+	jose "gopkg.in/square/go-jose.v2"
+)
+
+func TestTokenGeneratorKeys(t *testing.T) {
+
```

Comment:
for the tests, maybe we should is if/when/then comments since this is what we use in the other repositories?

Reply:
Let's do this in #24 next sprint.